Tech Lead/Minutes/20220623.md

```diff
@@ -0,0 +1,11 @@
+## Minutes of call 23 June 2022
+
+**Attendance**: all the usual suspects
```

Comment:
```suggestion
**Attendance**: all the usual suspects
**Attendance**: all the usual suspects (@bouweandela @bsolino @ehogan @hb326 @remi-kazeroni @schlunma @sloosvel @valeriupredoi @zklaus)
```
Feel free to add the github ids of the usual suspects, it might help those less familiar with our gang ;)